nvdaHelper/vbufBase/storage.cpp

```diff
@@ -912,19 +912,18 @@ int VBufStorage_buffer_t::getTextLength() const {
 	return length;
 }
 
-wstring  VBufStorage_buffer_t::getTextInRange(int startOffset, int endOffset, bool useMarkup) {
+bool VBufStorage_buffer_t::getTextInRange(int startOffset, int endOffset, wstring& text, bool useMarkup) {

```

Comment:
Can I ask about the order of the params here? I would expect out params like `text` to be last?.